production/helm/loki/templates/backend/service-backend-headless.yaml

```diff
@@ -0,0 +1,25 @@
+{{- $isSimpleScalable := eq (include "loki.deployment.isScalable" .) "true" -}}
+{{- if and $isSimpleScalable (not .Values.read.legacyReadTarget ) }}
+---
+apiVersion: v1
+kind: Service
```

Comment:
Why is this needed? I see a service below. Is there some duplication?

Reply:
I don't know k8s well enough to answer this, but I'm following the pattern of our other statefulsets that have both a service and a headless service. The headless service is used as the statefulset's `spec.serviceName`, whereas the other service is used for load-balancing.